course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -286,7 +286,6 @@ class CourseContainerViewModel(
                         }
                         if (isIAPFlow) {
                             if (isExpiredCoursePurchase) {
-                                eventLogger.upgradeSuccessEvent()
```

Comment:
Is there a specific reason we removed this? Removing it would prevent the Upgrade Success event from being triggered in the Expired Course scenario.

Reply:
reverted.